### DISCUSSION TITLE
How about a Markdown demo in Yellow?

### DISCUSSION THREAD

**Discussion by nibreh** [datenstrom/community#67] [Ask a question] [answered]
How about demo markdown syntax include in YellowCMS ?

If I suggest this, it's for beginners who have never learned about markdown (I was one of them). Once yellow is uploaded, you can found a tiny example page for introduce markdown. Very helpful for starting editing page just right after uploading Yellow.

It can be the page _about_ by default, or a PDF in media folder, or a link in the menu ...

**Comment by markseuffert** [comment]
Thanks Nibreh, that's a good idea. What would be a good place for a Markdown demo? What do you guys prefer, adding an example page to the installation or adding it to the [Yellow help](https://datenstrom.se/yellow/help/)?

**Comment by njoqi** [comment]
I would add a page to the documentation, and a link pointing to it in the default page.
This way, the Markdown demo can always be accessed and bookmarked for reference, but it allows new users to find it easily.

**Comment by BsNoSi** [comment]
I have created a markdown description in German. I can translate it into English, to be included in the download package. I will send it to Mark, as soon as it is available.

**Comment by Truckle** [comment]
Does Yellow support Github flavored markdown? And how about that cheat sheet? I'll take it in German if you don't have it in English.

**Comment by markseuffert** [comment]
I added a Markdown cheat sheet, a tiny one, feel free to contribute and add more information. :smile: 

https://github.com/annaesvensson/yellow-markdown#examples

**Comment by markseuffert** [comment, marked as answer]
I added a link to the Markdown cheat sheet. When editing a page in a web browser, there's a Markdown link in the right corner. This brings you to the Markdown syntax and shows some examples.

What do you think @nibreh, does this work?

**Comment by nibreh** [comment]
Hi. 

Yes it's working. This is very useful for beginners, who can now quickly learn to use yellow. And the new better installation makes it even easier to use.

Thanks !
